### PR TITLE
adding standard changes for logger

### PR DESCRIPTION
**Pull request by aradiquez**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/opentok/textchat-acc-pack/pull/197" target="_blank">ReviewNinja

File: js/opentok.js-text-chat/src/opentok-text-chat.js [hunk 1]
```diff
@@ -11,13 +11,15 @@
   var _logEventData = {
     // vars for the analytics logs. Internal use
     clientVersion: 'js-vsol-1.0.0',
-    componentId: 'textChatAccPack',
+    componentId: 'textchatAccPack',
```
Comment (Lucashuang0802) [inline js/opentok.js-text-chat/src/opentok-text-chat.js]: I see it's `textChatAccPack` in iOS and Android.

**Comment by Lucashuang0802** [comment]
In general, the mistake is: there should have matching success and failure action if attempt action is logged.

**Comment by Lucashuang0802** [comment]
@aradiquez pls rebase it.

**Pull request merged**